source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
+                  <p>
+                     <specList>
+                        <specDesc key="physLoc"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">physLoc</gi> element allows to encode any information related to or associated with the physical location of a bibliographic item. This includes, but is not limited to, the name of the holding institution, name or number of the building or room, any shelfmarks, etc. for the purpose of retrieval. The level of detail or machine readability of the encoding is generally at the discretion of the encoder and may vary depending on the information available. The <gi scheme="MEI">physLoc</gi> element may contain the following MEI-elements:</p>
+                  <p>
+                     <specList>
+                        <specDesc key="head"/>
+                        <specDesc key="repository"/>
+                        <specDesc key="identifier"/>
+                        <specDesc key="history"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">repository</gi> element is intended to contain a description of the institution or individual currently holding the bibliographic item. Its content is either extensive prose or structured, and thus better machine readable, information. The <gi scheme="MEI">history</gi> element on the other hand is a container for additional non-bibliographic details regarding the physical location of an item. It may contain, among others, one of the elements <gi scheme="MEI">acquisition</gi>, <gi scheme="MEI">exhibHist</gi> or <gi scheme="MEI">provenance</gi> to describe any events that coincided with a change of location like exhibitions, change of custody or alike.</p>
```

Comment:
```suggestion
                  <p>The <gi scheme="MEI">repository</gi> element is intended to contain a description of the institution or individual currently holding the bibliographic item. Its content is either extensive prose or structured, and thus better machine readable, information. The <gi scheme="MEI">history</gi> element on the other hand is a container for additional non-bibliographic details regarding the physical location of an item. It may contain the elements <gi scheme="MEI">acquisition</gi>, <gi scheme="MEI">exhibHist</gi> or <gi scheme="MEI">provenance</gi>, among others,  to describe any events that coincided with a change of location, such as exhibitions, change of custody, etc.</p>
```